app/services/copy_vacancy_asa_template.rb

```diff
@@ -45,9 +48,10 @@ def reset_legacy_fields(new_vacancy)
     end
 
     def current_steps(vacancy)
-      Publishers::Vacancies::VacancyStepProcess.new(:job_role,
-                                                    vacancy: vacancy,
-                                                    organisation: vacancy.organisation).steps
+      process = Publishers::Vacancies::VacancyStepProcess.new(:job_role,
+                                                              vacancy: vacancy,
+                                                              organisation: vacancy.organisation)
+      Publishers::VacancyFormSequence.new(vacancy: vacancy, organisation: vacancy.organisation, step_process: process).valid_steps
     end
   end
 end
```

Comment:
the unit test should reflect the new_vacancy.completed_steps fix

Reply:
We could probably change the name of this to completed_steps right?